components/builder-web/app/shared/project-settings/project-settings.component.html

```diff
@@ -1,12 +1,38 @@
 <div class="project-settings-component">
-  <div class="connect" *ngIf="!project && !connecting">
-    <button mat-raised-button color="primary" class="button" (click)="connect()" [disabled]="!hasPrivateKey">
-      Connect a plan file
-    </button>
-    <span class="no-keys" *ngIf="!hasPrivateKey">
-      <hab-icon symbol="add-circle"></hab-icon>
-      <a [routerLink]="['/origins', origin, 'keys']">Add required origin keys</a>
-    </span>
+  <h3>Default Visibility</h3>
+  <p>Set the default visibility of build artificats (.hart files) for the package. Applies to future build artifacts only. To change the visibility of the existing artifacts, see settings for each artifact in <a [routerLink]="['/pkgs', origin, name]">versions</a>.</p>
+  <hab-visibility-selector [setting]="visibility" (changed)="settingChanged($event)"></hab-visibility-selector>
+
+  <div class="note" *ngIf="!loadingInstallations && !gitHubAppInstalled">
+    <div class="note-icon">
+      <hab-icon symbol="github"></hab-icon>
+    </div>
+    <div class="note-text">
+      <p>Install the Builder Github app and allow access to the plan repository before connecting a new plan or building from the existing plans. After it's installed, <a (click)="refresh()">refresh</a> this page.</p>
+    </div>
+    <div class="note-cta">
+      <a href="{{ config['github_app_url'] }}" mat-raised-button color="primary" target="_blank">
+        Install GitHub App <hab-icon symbol="open-in-new"></hab-icon>
+      </a>
+    </div>
+  </div>
+
+  <h3>Connected Plans</h3>
+  <button mat-button mat-raised-button color="primary" class="button connect-btn" [matMenuTriggerFor]="menu" [disabled]="!hasPrivateKey || !gitHubAppInstalled">
+    <span class="connect-btn-text">Connect a plan file</span>
+    <hab-icon class="connect-btn-icon" symbol="drop-down"></hab-icon>
+  </button>
+  <mat-menu #menu="matMenu" [overlapTrigger]="false" class="connect-menu">
+    <button mat-menu-item (click)="openConnect('linux')">for Linux (kernel version 3.2 or later)</button>
+    <button mat-menu-item (click)="openConnect('linux2')">for Linux 2 (kernel version 2.6.32 or later)</button>
+    <button mat-menu-item (click)="openConnect('windows')">for Windows</button>
```

Comment:
Is it possible to make the option disabled from the list which has exiting plan connected? In the GIFT demo, after a Linux 3.2 or later plan is connected, the option seems to be still available.